packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -109,6 +120,29 @@ const PersonalDetailsForm = observer(() => {
         }
     }, [invalidate, is_language_changing]);
 
+    const hintMessage = ({ is_phone_number_editted, is_phone_number_empty }: THintMessage) => {
+        if (isPhoneNumberVerificationEnabled) {
+            if (account_settings?.phone_number_verification?.verified) {
+                return (
+                    <Localize
+                        i18n_default_text='To change your verified phone number, contact us via <0></0>.'
+                        components={[
+                            <OpenLiveChatLink
+                                text_size='xxs'
+                                key={0}
+                                className='account-form__fieldset--phone-verification-livechat-link'
+                            />,
+                        ]}
+                    />
+                );
+            } else if (is_phone_number_editted || is_phone_number_empty) {
```

Comment:
if phone number is empty, do we need to show "Save changes to enable verification." message?

Reply:
Yes we need to show this message when phone number is empty: https://www.figma.com/design/BhqkWGIHpm5UKFNGqEoks3/branch/SJ7X7uLxtQ2AeExS8nyTZ6/Release---v1.0.0---Account-settings?node-id=28046-39960&m=dev